doc/user/bmp.rst

```diff
@@ -93,6 +93,33 @@ There is one option that applies to the BGP instance as a whole:
 
    BMP Route Monitoring is not affected by this option.
 
+.. clicmd:: bmp startup-delay delay(0-4294967294)
+
+   This sets the delay (in milliseconds) after module startup
+   for BMP sessions to become active.
+
+   This setting is useful for BMP Monitoring because it allows BMP
+   to wait a bit for BGP to converge and the whole BGP state. It avoids
+   sending empty RIB content during synchronization on startup followed
+   by multiple incremental updates. It also avoids RIB-Out Pre-Policy Monitoring
+   to send duplicated messages on startup triggered by the reconfiguration of peer
+   policies.
+
+   This setting applies to all sessions defined in the same BGP Instance.
+   On module initialization (at BGP Startup), the time is recorded. Then,
+   established sessions will wait "delay" after this recorded time to start
+   sending BMP Mirroring, the initial BMP Monitoring synchronization and
+   following BMP Monitoring Messages containing incremental updates.
+
+   BMP Peer Up Messages are sent if the peer becomes available during this
+   period of time.
+
+   The startup delay is applied for BMP startup only. BMP Sessions configured
+   while the daemon is running will only wait if this initial timer has not expired
+   yet.
+
+   BMP Session Establishment is not affected by this option.
+
```

Comment:
is it the same meaning (line 121 and line 114? I think 121is more generic, because you may have flapping (peer down) messages.